data/styles/outdoors/include/priorities_4_overlays.prio.txt

```diff
@@ -978,7 +978,7 @@ tourism-artwork-painting                            # icon z15- (also has captio
 === 1550
 
 amenity-shelter                                     # icon z13- (also has caption(optional) z13-)
-amenity-shelter-public_transport                    # icon z13- (also has caption(optional) z13-)
+amenity-shelter-public_transport                    # icon z14- and caption z13 (also has caption(optional) z14-)
```

Comment:
also need to add `text: none;` to remove the z13 caption..

Reply:
Okay! Should be good now!